web/src/main.tsx

```diff
@@ -52,7 +52,9 @@ if (isProduction) {
 
 // Temporarily disabled to ensure we can more easily rollback
 // Also removes existing service workers to ensure they don't interfer
+
 if (navigator.serviceWorker) {
+  // eslint-disable-next-line unicorn/prefer-top-level-await
```

Comment:
As we plan on removing this code once we feel everything is stable I felt no need to "fix it" now by turning it into an await statment.